apps/src/templates/learn.html.ejs

```diff
@@ -8,6 +8,6 @@
   <a href="<%= root %>"><img id="learn-to-code" src="<%= assetUrl('media/promo.png') %>"></a>
   <a href="<%= root %>"><%= msg.watchVideo() %></a>
   <a href="<%= root %>"><%= msg.tryHOC() %></a>
-  <a href="<%= location.protocol + '//' + location.host %>"><%= msg.signup() %></a>
+  <a href="<%= location.protocol + '\/\/' + location.host %>"><%= msg.signup() %></a>
```

Comment:
This is a workaround for https://github.com/mde/ejs/issues/26 which is broken in [ejs](https://github.com/mde/ejs) 2.0.1 <= x <= 2.1.4, will be fixed in ~2.2.0